clang/test/3C/json_formating.c

```diff
@@ -0,0 +1,11 @@
+// RUN: 3c -base-dir=%S -alltypes -dump-stats -dump-intermediate -debug-solver %s
+// RUN: python -c "import json, glob; [json.load(open(f)) for f in glob.glob('*.json')]"
```

Comment:
Not so "easy": this is using the default working directory of `build/tools/clang/test/3C` and will fail if any invalid JSON file is somehow created there and not overwritten by the current test.  We should use something based on `%t`.  Unfortunately, I didn't look at this PR soon enough to raise an objection before it was merged, but other tests have similar problems, so we can follow up on all of the existing tests in #606.  Hopefully reviewers can remember to watch for this problem in future PRs.